### ISSUE TITLE
write multiple input loss function error

### ISSUE THREAD

**Issue by viet24dung** [keras-team/keras#13687]
Hello everybody!!
I just write Model test with multiple output and I want take it all and put it into loss function.
But when I write model code with 
`
model = Model([model.input, *y_true], [predic_label,L_loss])
`
predic_label and L_loss is output of lambda layer
shape:
   predic_label : (None,10)
   L_loss           : (None,1)
And I compile with 
`
def custom_Loss(y_true, y_pred):
    return y_pred
model.compile(optimizer=optimizer ,loss={"L_loss": custom_Loss})
`
I put breakpoint into "custom_Loss" function and I receive two times break in function "custom_Loss"

pls help me !!!!!
